.circleci/config.yml

```diff
@@ -1,11 +1,12 @@
+---
 version: 2.1
 orbs:
   prometheus: prometheus/prometheus@0.16.0
 executors:
   # Whenever the Go version is updated here, .promu.yml should also be updated.
   golang:
     docker:
-      - image: circleci/golang:1.17
+      - image: quay.io/prometheus/golang-builder:1.18-base
```

Comment:
We're actually switching these over to `cimg/go:1.18` except for `prometheus/prometheus`.